config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
+  devise_for :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root to: "home#index"
```

Comment:
Since we don't actually have a controller to handle a route named `home#index`, rails will raise an error. I believe we can just leave this out for now.